src/os_lib.eliom

```diff
@@ -102,6 +102,13 @@ module Email_or_phone = struct
         s, `Almost_email
       else if Re_str.string_match phone_regexp s 0 then
         let s = string_filter ((<>) ' ') s in
+        let s =
+          (* Also accept 00 prefix and normalize to + *)
```

Comment:
I could remove this for now, or leave it there as a small optimization for most Europeans (hard to imagine it causing trouble to anyone else). What do you think?

In the future we could get fancier and use geolocalization to guess the user's intention.